src/graphql/queries/events.ts

```diff
@@ -16,7 +16,7 @@ export const FETCH_ALL_EVENTS = gql`
 
 export const GET_ALL_EVENTS = gql`
   query {
-    events @client {
+    events {
```

Comment:
i'd prefer to keep the `@client` directive in - even though it's throwing an error right now, it makes it easier to find where we wanted to use it in the future when we want to address it
```suggestion
    events @client {
```